backend/src/server/routes/sanitizedSchemas.ts

```diff
@@ -27,6 +27,44 @@ export const integrationAuthPubSchema = IntegrationAuthsSchema.pick({
   updatedAt: true
 });
 
+export const DefaultResponseErrorsSchema = {
+  400: z.object({
+    statusCode: z.literal(400),
+    message: z.literal(
+      `<string>: Contains a human-readable message about what went wrong. 400 errors are bad request errors, and are will occur to happen when passing or unexpected bad data in the request`
+    ),
+    error: z.literal(`<string>: Contains a human-readable error name.`)
+  }),
+  404: z.object({
+    statusCode: z.literal(404),
+    message: z.literal(
+      `<string>: Contains a human-readable message about what went wrong. 404 errors are not found errors, and will occur when a resource is not found`
+    ),
+    error: z.literal(`<string>: Contains a human-readable error name.`)
+  }),
+  401: z.object({
+    statusCode: z.literal(401),
+    message: z.literal(
```

Comment:
https://github.com/Infisical/infisical/pull/2494/files#diff-f9f0d9293f47bd302107c64be06a2d477c681880ca2fac3f8f5dec10ae44a96eR34

@DanielHougaard The message may not be string, for validation error it will be JSON object. This would cause response schema internal server error

Reply:
Also @DanielHougaard You don't need to manually do this. There is a hook called `onHook` in fastify. You can write a plugin to inject it for each route.

Reply:
https://fastify.dev/docs/latest/Reference/Hooks/#onroute

Reply:
Yeah, this seems pretty redundant